health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/BulkUpload.js

```
@@ -0,0 +1,202 @@
+        />
+      )}
+      {fileData?.length > 0 && renderFileCards}
+      {showPreview && <XlsPreview file={fileUrl} onDownload={() => handleFileDownload(null, fileUrl)} onBack={() => setShowPreview(false)} />}
+      {showToast && (
+        <Toast
+          label={showToast?.label}
+          type={showToast?.key === "error" ? "error" : showToast?.key === "info" ? "info" : "success"}
+          // error={showToast?.key === "error" ? true : false}
+          isDleteBtn={true}
+          onClose={() => setShowToast(null)}
+        ></Toast>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)